Core/scripts/dirac-consumer.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python
+"""  This is a script to launch DIRAC consumers.
+"""
+
+import sys
+from DIRAC.ConfigurationSystem.Client.LocalConfiguration import LocalConfiguration
+from DIRAC import gLogger
+from DIRAC.Core.Base.ConsumerReactor import ConsumerReactor, loadConsumerModule
```

Comment:
You are missing the part which merges the specific error code of the extensions (```includeExtensionErrors```). Look at dirac-service for example